tests/test_ri_wellmod.py

```diff
@@ -257,3 +258,39 @@ def test_main_lgr_reek(tmpdir, mocker):
     ri_wellmod.main()
 
     assert Path(outfile).exists() and file_contains(outfile, "OP_1")
+
+
+# Test development version
+@pytest.mark.skipif(drogon_runpath() is None, reason="Could not find Drogon data")
```

Comment:
add
`@pytest.mark.skipif(not Path(RI_DEV).exists(), reason="No dev-version of ResInsight available")`

Reply:
Good point, done.